.github/ISSUE_TEMPLATE/epic-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Epic
+about: This template provides a basic structure for an epic issues.
+title: '[Epic] <title>'
+labels: Epic
+---
+
+### Goal
+
+_Introduction_
+
+### Success Metrics
+
+### Work items
```

Comment:
Doesn't GitHub have a tool to automagically link related, or in this case, child work items?

Reply:
I have not seen anything like that mentioned.  You can link PRs.  There is also Tracked Issues [https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists#navigating-tracked-issues](url), but neither is automatic